testtracer_test.go

```diff
@@ -0,0 +1,48 @@
+package opentracing
+
+// testTracer is a most-noop Tracer implementation that makes it possible for
+// unittests to verify whether certain methods were / were not called.
+type testTracer struct{}
```

Comment:
Oh, didn't even think about that. Good catch.

On Thu, Mar 10, 2016 at 10:42 PM Yuri Shkuro <notifications@github.com>
wrote:

> In testtracer_test.go
> <https://github.com/opentracing/opentracing-go/pull/72#discussion_r55784980>
> :
>
> > @@ -0,0 +1,48 @@
> > +package opentracing
> > +
> > +// testTracer is a most-noop Tracer implementation that makes it possible for
> > +// unittests to verify whether certain methods were / were not called.
> > +type testTracer struct{}
>
> I'd rather not have circular dependency, or in fact any extra dependencies
> in opentracing-go
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opentracing/opentracing-go/pull/72/files#r55784980>.
>